src/NoFrixion.MoneyMoov/Enums/AccountTypeEnum.cs

```diff
@@ -0,0 +1,37 @@
+﻿// -----------------------------------------------------------------------------
+//  Filename: AccountTypeEnum.cs
+// 
+//  Description: Provides an enumeration of account types. Mainly for fee calculations.
+// 
+//  Author(s):
+//  Axel Granillo (axel@nofrixion.com)
+// 
+//  History:
+//  16 07 2024  Axel Granillo   Created, Remote, Mexico City, Mexico.
+// 
```

Comment:
Are you impersonating Axel???

Reply:
Hehe. This was originally created by the great @agranillonf . I just moved it.

Reply:
Ah! Got it!

Reply:
oh stop it you 🙈